http-server/src/main/java/io/airlift/http/server/HttpServerChannelListener.java

```diff
@@ -91,11 +102,16 @@ public void onComplete(Request request)
         long beginToDispatchMillis = NANOSECONDS.toMillis((Long) request.getAttribute(REQUEST_BEGIN_TO_DISPATCH_ATTRIBUTE));
         long beginToEndMillis = NANOSECONDS.toMillis((Long) request.getAttribute(REQUEST_BEGIN_TO_END_ATTRIBUTE));
         logger.log(request,
```

Comment:
This is best done via the RequestLog mechanism.
See Server.setRequestLog(RequestLog)